Gemfile.lock

```diff
@@ -433,4 +433,4 @@ DEPENDENCIES
   websocket-client-simple!
 
 BUNDLED WITH
-   1.14.6
+   1.15.0
```

Comment:
Please remove these Gemfile.lock changes. See Eileen's contributing to Rails workshop around amending and rebasing for some help on how: https://youtu.be/7zoD6NZy6vY?t=1h25s